mypy/semanal_typeddict.py

```diff
@@ -363,7 +362,8 @@ def build_typeddict_typeinfo(self, name: str, items: List[str],
                                  types: List[Type],
                                  required_keys: Set[str],
                                  line: int) -> TypeInfo:
-        value_type = make_simplified_union(types)
+        value_type = UnionType(types)
```

Comment:
It is a good practice not to use `UnionType(types)`, see `UnionType.make_union`

Reply:
Thanks, fixed